### DISCUSSION TITLE
Ability to sort issues in the order as they appear on the board

### DISCUSSION THREAD

**Discussion by h0adp0re** [ankitpokhrel/jira-cli#200] [Ideas] [answered]
As a result of grooming and prioritizing issues, issues are ordered in a specific way on the board. It'd be nice to get the same impression of priorities on the CLI as I do on the web.

**Comment by ankitpokhrel** [comment, marked as answer]
This is done with #206 which adds a `--order-by` filter.

The list is sorted by `created` field and the sorting order is descending by default. However, you can use `--order-by` filter to list the issues sorted by `rank`.

```sh
# List issue in the same order as you see in the UI
$ jira issue list --order-by rank --reverse

# List epic issues order by the rank in ASC order
$ jira epic list KEY-1 --order-by rank --reverse

# List sprint issues order by the rank in ASC order
$ jira sprint list SPRINT_ID --order-by rank --reverse
```

**Reply by h0adp0re** [reply]
Thanks, much obliged.
